ramalama/cli.py

```diff
@@ -251,6 +251,9 @@ def login_parser(subparsers):
 
 
 def normalize_registry(registry):
+    # Determine the registry to use. Check the value of `RAMALAMA_TRANSPORT` env var if no `registry` argument is set.
+    registry = registry or os.getenv("RAMALAMA_TRANSPORT") or "ollama"
```

Comment:
This needs to take into account config('transport') rather then hard code ollama.